airflow/operators/bash_operator.py

```diff
@@ -78,7 +80,7 @@ def execute(self, context):
                     raise AirflowException("Bash command failed")
 
         if self.xcom_push:
-            return str(line.strip())
+            return str(line.strip(), sys.getdefaultencoding())
```

Comment:
no luck here, format is giving me the same result as str with python3
```python
>>> '{}'.format(b'text')
"b'text'"
```
I could not find a better way to do it that is workling well with python2 and 3

There may be another way with decode
```python
'{}'.format(b'text'.decode('utf-8'))
```
```python
'{}'.format(b'text'.decode(sys.getdefaultencoding()))
```
Any other idea ?